Networking/WebSockets/BuiltInWebSocket.cc

```diff
@@ -184,6 +184,25 @@ namespace litecore { namespace websocket {
         // Create the HTTPLogic object:
         Dict headers = options()[kC4ReplicatorOptionExtraHeaders].asDict();
         HTTPLogic logic {Address(url()), Headers(headers)};
+        bool foundUserAgent = false;
+        for (auto iter = headers.begin(); iter != headers.end(); ++iter) {
+            if (iter.keyString().caseEquivalent("User-Agent")) {
+                foundUserAgent = true;
+                break;
+            }
+        }
+        if (!foundUserAgent) {
+            string ua = "couchbase-lite-core/";
+            alloc_slice ver = c4_getVersion();
```

Comment:
I think we can just put whatever returned here without discarding the info -The CBL platforms include this info to the User-Agent as well.